src/archiefbeheercomponent/conf/locale/nl/LC_MESSAGES/django.po

```diff
@@ -476,90 +550,180 @@ msgstr ""
 "{{ uuid }}, {{ bronorganisatie }} en {{ identificatie }}. Bijvoorbeeld: "
 "https://gemeente.lan/mijnzaken/zaak/{{ uuid }}"
 
-#: archiefvernietigingscomponent/destruction/models.py:393
+#: archiefbeheercomponent/destruction/models.py:416
+msgid "days until reminder"
+msgstr "dagen tot herinnering"
+
+#: archiefbeheercomponent/destruction/models.py:419
+msgid ""
+"Number of days until an email is sent reminding that the list needs to be "
+"dealt with"
+msgstr ""
+"Aantal dagen tot een email is verzonden als herinnering dat een actie is vereist voor een lijst"
+
+#: archiefbeheercomponent/destruction/models.py:425
+msgid "create zaak"
+msgstr "maak zaak aan"
+
+#: archiefbeheercomponent/destruction/models.py:428
+msgid "Should a case be created once a destruction list has been deleted?"
+msgstr "Moet een zaak aangemaakt worden als een vernietigingslijst wordt verwijderd?"
+
+#: archiefbeheercomponent/destruction/models.py:432
+msgid "source organisation"
+msgstr "bronorganisatie"
+
+#: archiefbeheercomponent/destruction/models.py:435
+msgid "Source organisation RSIN"
+msgstr "bronorganisatie RSIN"
```

Comment:
```suggestion
msgstr "Bronorganisatie RSIN"
```

Although I prefer all lowercase labels in models, translations should match the original text.